cirq-core/cirq/type_workarounds.py

```diff
@@ -12,7 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Module defining NotImplementedType."""
+"""Workarounds for python typing gotchas."""
```

Comment:
What's the technical term for a "gotcha"?

Reply:
reworded.  "gotcha" is likely not a great word